doc: add v2 backing image doc (#1031)
longhorn/website: content/docs/1.8.0/advanced-resources/system-backup-restore/backup-longhorn-system.md

- StorageClasses
- Volumes

> > **Node:** Longhorn does not back up v2 data engine backing images as it is not supported.

Reviewer: ```suggestion
> **Note:** Longhorn is unable to back up V2 Data Engine backing images.
```

Author: fixed.
